man/man4/zfs.4

```diff
@@ -2275,6 +2275,14 @@ If
 .Sy 0 ,
 generate a system-dependent value close to 6 threads per taskq.
 .
+.It Sy zio_taskq_wr_iss_ncpus Ns = Ns Sy 32 Pq uint
+Determines the number of CPUs to run write issue taskqs.
+.Pp
+While an optimal value will be system dependent, a suggested value
+is the number of actual CPUs in the system, divided by the
+.Sy spa_num_allocators
+value.
```

Comment:
Why not default this to the recommended value, something like `MAX(boot_ncpus / spa_num_allocators, 1)`.

Reply:
Sounds good, I've added to the latest rev.

Reply:
Requested change is implemented in spa_taskqs_init():

	case ZTI_MODE_SYNC:

		/*
		 * Create one wr_iss taskq for every 'zio_taskq_wr_iss_ncpus',
		 * not to exceed the number of spa allocators.
		 */
		if (zio_taskq_wr_iss_ncpus == 0) {
			count = MAX(boot_ncpus / spa->spa_alloc_count, 1);
		} else {
			count = MAX(1,
			    boot_ncpus / MAX(1, zio_taskq_wr_iss_ncpus));
		}
		count = MAX(count, (zio_taskq_batch_pct + 99) / 100);
		count = MIN(count, spa->spa_alloc_count);